rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -296,36 +296,42 @@ MasterGraph::build()
     start_processing();
     return Status::OK;
 }
-Image *
-MasterGraph::create_loader_output_image(const ImageInfo &info)
+
```

Comment:
Remove the extra line

Reply:
Introduced the line to basically have a single line space between 2 API definitions